cmd/kops/create_cluster.go

```diff
@@ -488,6 +488,9 @@ func RunCreateCluster(ctx context.Context, f *util.Factory, out io.Writer, c *Cr
 	if c.ContainerRuntime != "" {
 		cluster.Spec.ContainerRuntime = c.ContainerRuntime
 	}
+	if cluster.Spec.ContainerRuntime == "containerd" && cluster.Spec.Networking.Kubenet != nil {
```

Comment:
There are a couple of other things to test for here too. `Networking.Kopeio` and `Networking.External`. Maybe use `components.UsesCNI(cluster.Spec.Networking) `?

Reply:
Thanks. I think this should do it.